eng/common/scripts/Helpers/git-helpers.ps1

```diff
@@ -247,4 +274,93 @@ function Write-RateLimit {
   Write-Host "    remaining=$($RateLimit.remaining)"
   Write-Host "    reset=$($RateLimit.reset)"
   Write-Host ""
-}
\ No newline at end of file
+}
+
+function GetAIReviewThreads {
+  param(
+    [string]$repoOwner,
+    [string]$repoName,
+    [string]$prNumber,
+    [array]$reviewers
+  )
+
+  $reviewers ??= @('copilot-pull-request-reviewer')
+
+  $reviewThreadsQuery = @'
+query ReviewThreads($owner: String!, $name: String!, $number: Int!) {
+  repository(owner: $owner, name: $name) {
+    pullRequest(number: $number) {
+      reviewThreads(first: 100) {
+        nodes {
+          id
+          isResolved
+          comments(first: 100) {
+            nodes {
+              body
+              author {
+                login
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+'@
+
+  $variables = @{ owner = $repoOwner; name = $repoName; number = [int]$prNumber }
+  $response = RequestGithubGraphQL -query $reviewThreadsQuery -variables $variables
+  $reviews = $response.data.repository.pullRequest.reviewThreads.nodes
+
+  $threadIds = @()
+  # There should be only one threadId for copilot, but make it an array in case there
+  # are more, or if we want to resolve threads from multiple ai authors in the future.
+  # Don't mark threads from humans as resolved, as those may be real questions/blockers.
+  foreach ($thread in $reviews) {
+    if ($thread.comments.nodes | Where-Object { $_.author.login -in $reviewers }) {
+      $threadIds += $thread.id
+      continue
+    }
+  }
+
+  return $threadIds
+}
+
+function TryResolveAIReviewThreads {
+  param(
+    [string]$repoOwner,
+    [string]$repoName,
+    [string]$prNumber,
+    [array]$reviewers
+  )
+
+  $resolveThreadMutation = @'
+mutation ResolveThread($id: ID!) {
+  resolveReviewThread(input: { threadId: $id }) {
+    thread {
+      isResolved
+    }
+  }
+}
+'@
+
+  $threadIds = GetAIReviewThreads -repoOwner $repoOwner -repoName $repoName -prNumber $prNumber -reviewers $reviewers
+
+  if (!$threadIds) {
+    return $false
+  }
+
+  if ($SkipMerge) {
```

Comment:
Did you intend to look at this as a global variable?

Reply:
Another variable lost in the refactor :)